main.go

```diff
@@ -0,0 +1,605 @@
+package main
```

Comment:
Sorry for a bit late comment on this but better late than sorry :D I would suggest renaming the repo to not define the API protocol, so maybe just flow-evm-api or flow-evm-gateway, since I'm pretty sure down the line there might be other protocols that will be required for this to work well with clients.

Reply:
ok will rename the repo